dist/ansible/scripts/ovn-setup.sh

```diff
@@ -23,9 +23,8 @@ is_cluster_up () {
 }
 
 is_api_server_ready () {
-  # kubectl cluster-info leaves ^[[0 in output string
-  #apiserver=$(kubectl cluster-info | gawk '/Kubernetes master/{ print $6 }')
-  apiserver=$(oc whoami --show-server)
+  #apiserver=$(oc whoami --show-server)
+  apiserver=$(kubectl config current-context | cut -f 2 -d/)
```

Comment:
@pecameron I was curious about this, so I tried it with docker-in-docker and I get "dind-cluster".  What are we supposed to get on a daemonset install?

Was there something sub-optimal with the kubectl cluster-info?  An alternative might be `kubectl config view -o jsonpath='{.clusters[0].cluster.server}'`

Reply:
On netdev22
# kubectl config view -o jsonpath='{.clusters[0].cluster.server}'
https://wsfd-netdev22.ntdv.lab.eng.bos.redhat.com:8443

kubectl cluster-info output terminal formatting sequences ^[[ along with the string. 
^[[0;33mhttps://wsfd-netdev22.ntdv.lab.eng.bos.redhat.com:8443^[[0m

Ben suggested the current line. I can change it if you prefer your line.

Reply:
@dcbw Made the change you suggested, PTAL